### ISSUE TITLE
when trying to build the docker-compose-file with `podman compose` **password_manager** give some cryptic error [BUG]

### ISSUE THREAD

**Issue by 0hmX** [frankhart2018/primistore#95]
**Describe the bug**
when trying to build the docker-compose-file with `podman compose` **password_manager** give some cryptic error, I suspect this error is also occurring when building with docker, but I have not verified.

**To Reproduce**
1. Git clone the repository
2. Run `IP=<IP> REACT_APP_PI=<true/false> LOCAL_DIR=<charset-dir-path> podman compose --file docker-compose.yml up`

**Expected behavior**
Dose not generate error!

**Screenshots**
![image](https://github.com/frankhart2018/primistore/assets/109351887/f6025b7b-dd8b-4b4d-a0e7-31dced8dcd97)

**Desktop (please complete the following information):**
 - OS: Fedora
 - Version: 39
```
[password_manager] | node:internal/modules/cjs/loader:1227
[password_manager] |   throw err;
[password_manager] |   ^
[password_manager] | 
[password_manager] | Error: Cannot find module '/app/app.js'
[password_manager] |     at Module._resolveFilename (node:internal/modules/cjs/loader:1224:15)
[password_manager] |     at Module._load (node:internal/modules/cjs/loader:1050:27)
[password_manager] |     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:168:12)
[password_manager] |     at node:internal/main/run_main_module:30:49 {
[password_manager] |   code: 'MODULE_NOT_FOUND',
[password_manager] |   requireStack: []
[password_manager] | }
[password_manager] | 
[password_manager] | Node.js v22.1.0
```

**Comment by frankhart2018** [comment]
This does not seem like enough information to figure out the issue. Are there any other important logs that's missing?

**Comment by frankhart2018** [comment]
Found the issue, part of password-manager was moved to typescript, and it wasn't compiled in the service's Dockerfile. Fixed that as part of #54. Closing this. Thanks for reporting the issue.
